Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/tests/ActionsTests.cs

```diff
@@ -0,0 +1,251 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Adapters;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Dialogs.Adaptive;
+using Microsoft.Bot.Builder.Dialogs.Declarative;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Schema;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Tests
+{
+    [TestClass]
```

Comment:
Should we migrate these tests to Xunit?

Reply:
Not for now, this is how composer spits out the runtime, I want to minimize changes to composer to make it easier to maintain.